src/neo4j/neo4j-document.js

```diff
@@ -166,3 +169,32 @@ export async function addDocumentToNeo4j(doc) {
 
   return;
 }
+
+export async function addAllDocumentsToNeo4j() {
+  let rdbConn;
+  let testDb;
```

Comment:
Here's where we need to have a configurable database instance that could access  non-test (production) data.

 In practice, probably need to hook into those used by a running factoid server instance: `db` in `src/server/db.js` which uses environment variables in `src/config.js` which could be overwritten at runtime.